### PR TITLE
Update skill names; use “Disziplin” in German

### PR DESCRIPTION
**Pull request by JonasWanke**

### CODE REVIEW

**Review by Benjamin-Frost — APPROVED** [review]

File: src/components/Header.astro [hunk 1]
```diff
@@ -11,11 +11,11 @@ const t = useTranslations(lang);
 
 const skills = [
   {
-    name: "Mobile",
+    name: "App-Entwicklung",
```
Comment (Benjamin-Frost) [inline src/components/Header.astro:14]: Needs to be translated for english version of the page.

File: src/components/Header.astro [hunk 2]
```diff
@@ -11,11 +11,11 @@ const t = useTranslations(lang);
...
     href: getRelativeLocaleUrl(lang, "mobile-applications-development"),
   },
   {
-    name: "Full-Stack",
+    name: "Software-Entwicklung",
```
Comment (Benjamin-Frost) [inline src/components/Header.astro:18]: Needs to be translated for english version of the page.

**Review by Benjamin-Frost — CHANGES_REQUESTED** [review]

File: src/components/dropdowns/Dropdown.tsx [hunk 3]
```diff
@@ -49,12 +49,12 @@ export function Dropdown<T>({
...
               leaveTo="opacity-0"
             >
-              <ListboxOptions className="absolute z-10 mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
+              <ListboxOptions className="absolute z-10 mt-1 max-h-60 overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
                 {options.map((option, index) => (
                   <ListboxOption
                     key={index}
                     className={
-                      "relative cursor-pointer select-none py-2 pl-3 pr-9 text-gray-900 hover:bg-wsg-orange-700"
+                      "relative cursor-pointer select-none px-4 py-2 text-gray-900 hover:bg-wsg-orange-700"
```
Comment (Benjamin-Frost) [inline src/components/dropdowns/Dropdown.tsx:57]: I can't explain why, but somehow it looked better before.

Comment (JonasWanke) [reply src/components/dropdowns/Dropdown.tsx]: I found it a bit weird here:

image

But I get what you mean in the other places and here in German, so I reverted it

**Review by Benjamin-Frost — APPROVED** [review]

**Pull request merged**